images/linux/scripts/installers/nvm.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+################################################################################
+##  File:  nvm.sh
+##  Desc:  Installs Nvm
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+export NVM_DIR="/etc/skel/.nvm"
+# should create directory with full permissions.
```

Comment:
It is still not clear why we need full permissions here and below, could you please clarify and add a comment?

Reply:
Yes, It was previously added when nvm was installed to /opt/cause it's system dir. Nvm installs new node versions to its dir. But now we install It to skel and nvm dir copies to $HOME dir of user, so I removed this action. I'll rerun my checks if everything is ok I'll leave a comment and close conversation.

Reply:
@sergey-akhalkov , removed full permissions and work as expected